include/system_key.h

```diff
@@ -0,0 +1,13 @@
+#ifndef SYSTEM_KEY_INCLUDED
```

Comment:
Some sort of Percona lawyer thing comment

Reply:
Done